R/data_read.R

```diff
@@ -96,50 +116,76 @@ data_read <- function(path, path_catalog = NULL, encoding = NULL, verbose = TRUE
     utils::unzip(path, exdir = d)
     path <- file.path(d, dest[1])
   } else {
-    stop("The zip-file does not contain any supported file types.", call. = FALSE)
+    insight::format_error("The zip-file does not contain any supported file types.")
   }
 
   path
 }
 
 
+.skip_empty_columns <- function(out, verbose) {
+  # find empty variables
```

Comment:
There is already `datawizard::remove_empty_columns()`.  The difference is that it doesn't remove columns with only `""`, maybe we could modify it instead of making a new helper?

Reply:
Actually I'm not sure it's worth adding an argument here when the user could just do `data_read(...) |> remove_empty_columns()`

Reply:
yeah, true. Forgot we have those functions ;-)